packages/rpc-engine/middlewares/call-rpc.js

```diff
@@ -21,10 +21,10 @@ const RequestLockMethods = [
   'wallet_switchEthereumChain',
   'wallet_addConfluxChain',
   'wallet_addEthereumChain',
-  'cfx_sendTransaction',
-  'personal_sign',
-  'cfx_signTypedData_v4',
-  'eth_signTypedData_v4',
+  // 'cfx_sendTransaction',
```

Comment:
these methods will be not required to be locked when interacting with them?

Reply:
Method in this array will trigger unlock your if called while client is locked.

Reply:
but we removed these 4 methods, for example, method `cfx_sendTransaction` should trigger the unlock UI when fluent is locked

Reply:
these four methods should have the correct address in params
since wallet won't give them user address when locked
these four methods should be blocked